mpl_interactions/generic.py

```diff
@@ -507,8 +509,19 @@ def hyperslicer(
     figsize=None,
     display=True,
     force_ipywidgets=False,
+    play_buttons=False,
+    play_button_pos='right',
     **kwargs,
 ):
+    
+    arr = np.asarray(np.squeeze(arr))
+    if not np.atleast_2d(arr):
+        warnings.warn('Invalid number of dimensions for imshow')
+        break
```

Comment:
So this `break` won't do anything and `np.at_least2d` returns an array no a boolean.  I think better is `raise` a valueerrror
```suggestion
    if not arr.ndim > 2:
        raise ValueError(f"arr must be at least 3D, but it is {arr.ndim}D")
```

Reply:
yeah this last commit was a disaster lol working on it now.

Reply:
Should it raise error for 2d arrays or just work as imshow?

Reply:
idk. it seems silly to have it just become `plt.imshow` but on the other hand its an arbitrary resitrction

Reply:
Well require >= 3d for now at least. Ill add "Use mpl_interactions.imshow for image data" to the value error